pyanaconda/modules/storage/bootloader/installation.py

```diff
@@ -98,6 +98,10 @@ def run(self):
             log.debug("The bootloader configuration is disabled.")
             return
 
+        if self._payload_type in PAYLOAD_LIVE_TYPES and \
+           isinstance(self._storage.bootloader, SystemdBoot):
+            raise BootloaderInstallationError("systemd-boot, not supported on live media.")
```

Comment:
maybe `"systemd-boot is not supported on live media."`?

Reply:
Sure.